src/processors.zig

```diff
@@ -181,6 +179,46 @@ pub const CrossPageLinkProcessor = struct {
             }
         }
     }
+    fn parseAltText(
+        pctx: anytype,
+        alt_text: []const u8,
+        maybe_scale: ?[]const u8,
+    ) !void {
+        if (maybe_scale) |scale| {
+            if (parseResolutionFromAlt(scale)) |res| {
```

Comment:
nitpick: should be renamed, I think, to just `parseResolution`, since `scale` isn't really part of alt text